bucoffea/monojet/definitions.py

```diff
@@ -965,9 +986,9 @@ def photon_trigger_sf(weights, photons, df):
     if year == 2016:
         sf =  np.ones(df.size)
     elif year == 2017:
-        sf = sigmoid(x,0.335,217.91,0.065,0.996) / sigmoid(x,0.244,212.34,0.050,1.000)
+        sf = sigmoid(x,1.140,219.12,0.086,0.996) / sigmoid(x,0.171,207.22,0.092,1.000)
```

Comment:
Can you add an if block here to differentiatae UL vs non-UL? I think the way it is written now, it changes the values for non-UL monojet, too

Reply:
Thanks, this is done in eb0a359b041f338f937d09b54d0d365116173bb7